lib/cargoDeny.nix

```diff
@@ -0,0 +1,27 @@
+{ cargo-deny
+, lib
+, mkCargoDerivation
+}:
+
+{ cargoDenyExtraArgs ? ""
+, cargoDenyChecks ? ["bans" "licenses" "sources"]
```

Comment:
We should keep this as a string (and not a list of strings). The caller can format it however they'd like but it simplifies things to not need to wrap into a list if there's just one value to pass in